servicetalk-log4j2-mdc-utils/src/main/java/io/servicetalk/log4j2/mdc/utils/ServiceTalkThreadContextMap.java

```diff
@@ -40,16 +39,18 @@
  */
 public class ServiceTalkThreadContextMap implements ReadOnlyThreadContextMap, CleanableThreadContextMap {
     private static final Key<Map<String, String>> key = Key.newKey("log4j2Mdc");
+    @SuppressWarnings("RedundantStringConstructorCall")
+    private static final String NULL_STRING = new String("");
 
     @Override
     public final void put(String key, String value) {
-        getStorage().put(key, value);
+        getStorage().put(key, wrapNull(value));
```

Comment:
Since MDC doesn't have a contains operator, why not just call remove(key) here? Is the concern that getCopyOfContextMap wouldn't show the null valued key?